controllers/grid/settings/user/form/UserForm.php

```diff
@@ -62,12 +64,24 @@ public function initData()
 
             foreach ($userGroups as $userGroup) {
                 $userGroupIds[] = $userGroup->id;
-                $masthead[$userGroup->id] = Repo::userGroup()->userOnMasthead($this->userId, $userGroup->id);
             }
+
+            // Get user group IDs for user groups this user should not be displayed on the masthead for
+            // The others will be selected per default
+            $notOnMastheadUserGroupIds = UserUserGroup::query()
+                ->withUserId($this->userId)
+                ->withActive()
+                ->withMastheadOff()
+                ->get()
+                ->map(
+                    fn (UserUserGroup $userUserGroup) => $userUserGroup->user_group_id
```

Comment:
Does the `HasCamelCasing` trait not allow the use of `userGroupId` here? `lower_camel_case` columns should be abstracted by the container level.

Reply:
Oh, yes, my fault... Corrected...